tryton.cfg

```diff
@@ -0,0 +1,10 @@
+[tryton]
+version=5.2.0
+depends:
+    party
+    analytic_bank_statement
+    account_bank_statement
+    account_bank_statement_account
+    analytic_account
```

Comment:
This can be reduced to
```
analytic_bank_statement
account_bank_statement_account
```
anything else will already be installed due to the dependency tree